README.md

```diff
@@ -366,6 +366,9 @@ Options:
   --caldav-passwd, --caldav-passwd-pass-path TEXT
                                   Path in the UNIX password manager to fetch
                                   the caldav password from
+  --caldav-passwd-cmd CMD
+				  A command-line (which is run in a shell) that
```

Comment:
nit: Let's change `command-line` to `command`. :)

Reply:
Done. (I also noticed I had accidentally used TAB instead of SPACE here. Fixed now.)